### PR TITLE
python 2 to 3

### PR DESCRIPTION
**Pull request by akshayhavile**
Moving repo to python 3.
Installed on vmware DU to check getting installed properly or not.

### CODE REVIEW

File: pf9_saml_auth/v3/base.py [hunk 1]
```diff
@@ -15,7 +15,7 @@
 """BasePF9SAMLPlugin class."""
 
 import base64
-import urlparse
+from six.moves import urllib
```
Comment (poojaghumre) [inline pf9_saml_auth/v3/base.py:18]: nit: from six.moves.urllib import parse

and change line80 to "parse.urlparse"

File: pf9_saml_auth/v3/okta.py [hunk 2]
```diff
@@ -14,7 +14,7 @@
 
 """Okta SAML authentication driver."""
 import re
-import urlparse
+from six.moves import urllib
```
Comment (poojaghumre) [inline pf9_saml_auth/v3/okta.py:17]: same here and other places

File: setup.py [hunk 3]
```diff
@@ -50,8 +52,9 @@ def read(fname):
         'lxml',
         'oktaauth',
         'python-keystoneclient',
+        'six>=1.10.0',
     ],
-    python_requires='>=2.7, !=3.*',
+    python_requires='>=2.7',
```
Comment (poojaghumre) [inline setup.py:57]: Have you verified that this is compatible with both python2.7 and python3.6?

**Review by poojaghumre — COMMENTED** [review]
Thanks Akshay!

File: pf9_saml_auth/v3/base.py (hunk 1 above)
Comment (akshayhavile) [reply pf9_saml_auth/v3/base.py:18]: will change that.

File: pf9_saml_auth/v3/okta.py (hunk 2 above)
Comment (akshayhavile) [reply pf9_saml_auth/v3/okta.py:17]: will change that

File: setup.py (hunk 3 above)
Comment (akshayhavile) [reply setup.py:57]: Tested on python2.7, didn't get python3.6 env.

**Comment by shahpratikr** [comment]
discarding this changes

**Pull request closed without merging**